src/api/voting.ts

```diff
@@ -98,23 +97,17 @@ export type IProcessState = {
 }
 
 /** Contains a summary of the most relevant process details */
-export type IProcessSummary = Pick<IProcessState,
-    "entityId" |
-    "status" |
-    "startBlock" |
-    "endBlock" |
-    "envelopeType" |
-    "entityIndex"
-> & {
-    /** The amount of votes registered */
-    envelopeHeight: number
-    /** The IPFS URI pointing to the JSON metadata file */
-    metadata: string
-    /** The origin from which the process was created */
-    sourceNetworkId: ISourceNetworkId
-}
+export type ProcessSummary =
+    Pick<ProcessState, "entityId" | "status" | "startBlock" | "endBlock" | "envelopeType" | "entityIndex"> & {
+        /** The amount of votes registered */
+        envelopeHeight: number
+        /** The IPFS URI pointing to the JSON metadata file */
+        metadata: string
+        /** The origin from which the process was created */
+        sourceNetworkId: keyof typeof SourceNetworkId
```

Comment:
@brickpop  I understand that it is tested that the resulting sourcenetworkID enum is same as before (in order to have backwords compatibility)

Reply:
It is :)
The keys were initially hardcoded, but now this allows to automatically have the list of keys being taken from the Protobuf model